x-pack/solutions/security/plugins/security_solution/public/management/pages/policy/view/policy_settings_form/components/advanced_section.test.tsx

```diff
@@ -20,12 +20,12 @@ import { AdvancedPolicySchema } from '../../../models/advanced_policy_schema';
 import { within } from '@testing-library/react';
 import { set } from '@kbn/safer-lodash-set';
 
+jest.setTimeout(15 * 1000); // Costly tests, hitting 2 seconds execution time locally
```

Comment:
Ran locally 50 times with no failures. As commented, some tests in this file hit 2 seconds execution time, it is likely that the default 5 seconds is not enough for it to complete in CI. Hence changing the default timeout.